src/commands/Minion/k.ts

```diff
@@ -1,6 +1,57 @@
-import { CommandStore, KlasaMessage } from 'klasa';
+import { calcWhatPercent, increaseNumByPercent, objectKeys, reduceNumByPercent, round } from 'e';
+import { CommandStore, KlasaMessage, KlasaUser } from 'klasa';
 
+import { Activity } from '../../lib/constants';
+import killableMonsters from '../../lib/minions/data/killableMonsters';
+import { minionNotBusy, requiresMinion } from '../../lib/minions/decorators';
+import { AttackStyles, resolveAttackStyles } from '../../lib/minions/functions';
+import calculateMonsterFood from '../../lib/minions/functions/calculateMonsterFood';
+import findMonster from '../../lib/minions/functions/findMonster';
+import reducedTimeFromKC from '../../lib/minions/functions/reducedTimeFromKC';
+import removeFoodFromUser from '../../lib/minions/functions/removeFoodFromUser';
+import { calcPOHBoosts } from '../../lib/poh';
 import { BotCommand } from '../../lib/structures/BotCommand';
+import { MonsterActivityTaskOptions } from '../../lib/types/minions';
+import {
+	addArrayOfNumbers,
+	formatDuration,
+	isWeekend,
+	itemNameFromID,
+	randomVariation,
+	removeDuplicatesFromArray
+} from '../../lib/util';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+const invalidMonster = (prefix: string) =>
+	`That isn't a valid monster, the available monsters are: ${killableMonsters
+		.map(mon => mon.name)
+		.join(', ')}. For example, \`${prefix}minion kill 5 zulrah\``;
+
+const { floor } = Math;
+
+function applySkillBoost(
+	user: KlasaUser,
+	duration: number,
+	styles: AttackStyles[]
+): [number, string] {
+	const skillTotal = addArrayOfNumbers(styles.map(s => user.skillLevel(s)));
+
+	let newDuration = duration;
+	let str = '';
+	let percent = round(calcWhatPercent(skillTotal, styles.length * 99), 2);
+
+	if (percent < 50) {
+		percent = 50 - percent;
```

Comment:
so -50% penalty if less than 50 but max +15% if more than 50?

Seems wrong since you said you wanted just -15% to 15%+